### ISSUE TITLE
Ensure documentation, especially images, are up to date

### ISSUE THREAD

**Issue by vengroff** [vengroff/censusdis#135]
In the `README.md` and in the docs that end up on https://censusdis.readthedocs.io/en/latest/index.html#, there are a number of embedded images of maps that are linked to the notebooks that produced them. Some of these are likely out of date as they were produced with older version of `censusdis`. Newer versions may produce somewhat different images.

The goal of this issue is to audit the images and the notebooks they are linked to, identify mismatches, and where necessary, replace the images in documentation with the images currently being produced by the notebooks.

**Comment by ibzimh** [comment]
Hi! Can I give this a shot?

**Comment by vengroff** [comment]
Yes, I would be happy to have you take a look at it. Do you want to start with a small pieces with just one or two images and then do a pull request for that so we can both be sure you are on the right track?

**Comment by vengroff** [comment]
Please have a look at https://github.com/vengroff/censusdis/blob/main/DEVELOPER.md for the basics on working on this project.

**Comment by ibzimh** [comment]
Hi, I've submitted a pull request with some changes.

**Comment by vengroff** [comment]
Thank you very much for your contribution, @ibzimh. Will definitely let you know as more issues come up.
